secator/cli_helper.py

```
@@ -39,7 +39,19 @@
 	'dry_run': {'is_flag': True, 'short': 'dr', 'default': False, 'help': 'Enable dry run'},
 	'sync': {'is_flag': True, 'help': 'Run tasks locally or in worker', 'opposite': 'worker'},
 	'no_poll': {'is_flag': True, 'short': 'np', 'default': False, 'help': 'Do not live poll for tasks results when running in worker'},  # noqa: E501
-	'process': {'is_flag': True, 'short': 'nps', 'default': True, 'help': 'Enable secator processing', 'reverse': True},
+	'process': {'is_flag': True, 'short': 'ps', 'default': True, 'help': 'Enable secator processing', 'reverse': True},
```

Pylint (3.3.7)

[convention] Line too long (116/100)

(C0301)